gradle/libs.versions.toml

```diff
@@ -36,7 +36,7 @@ guava = { module = "com.google.guava:guava", version = "33.3.1-jre" }
 
 kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
 kotlinpoet-ksp = { module = "com.squareup:kotlinpoet-ksp", version = "2.0.0" }
-kotlin-compile-testing-ksp = { module = "com.github.tschuchortdev:kotlin-compile-testing-ksp", version = "1.6.0" }
+kotlin-compile-testing-ksp = { module = "dev.zacsweers.kctfork:ksp", version = "0.7.0" }
```

Comment:
ended up moving to a fork from zacsweers since it is more regularly updated to newer kotlin versions.

The previous version was compiling against 1.9 which broke the 2.1 upgrade